sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeContentOperation.cs

```diff
@@ -128,5 +151,36 @@ private static FormPageCollection ConvertValue(IReadOnlyList<PageResult_internal
 
             return new FormPageCollection(pages);
         }
+
+        private async ValueTask<RequestFailedException> CreateExceptionForFailedOperationAsync(bool async, IReadOnlyList<FormRecognizerError> errors)
```

Comment:
It could be worth putting this method somewhere else so we don't repeat code. Error handling could change or be improved at some point in the future, so we could end up missing changes. We already have an `OperationExtensions` class, so it would be a good place to put it as an `internal static` method.

Reply:
I thought about it, but wanted to keep it per operation in case in the future an endpoint decides to either add more to the errors, change structure, or the behavior change and we need to do some customization around it. What do you think?

Reply:
I think your concern is reasonable, but I wonder what's the likelihood of changes like that happening in the near future (specially because we don't expect big structure changes after GA). Even in that scenario, I think we should have the shared logic in a single place, and the customized bits hidden in their corresponding types (it depends a lot on the kind of change we will have, though).

I don't feel too strongly about it, though, so I wouldn't call it blocking if you think that keeping them separated may be worth it.

Reply:
Maybe using the `ClientCommon` class for that? We're only using it for validating the model id, for now. If `ClientCommon` doesn't feel right, maybe renaming it?